pages/send/PaymentSuccess.tsx

```diff
@@ -30,6 +36,31 @@ export function PaymentSuccess() {
             <Text className="text-2xl text-muted-foreground font-semibold2">{getFiatAmount(+amount)}</Text>
           }
         </View>
+        {lnurlSuccessAction &&
+          <View className="flex flex-col gap-2 items-center">
+            <Text className="text-muted-foreground text-center font-semibold2">
+              Message From Receiver
+            </Text>
+            {lnurlSuccessAction.tag == "message" &&
+              <Text className="text-foreground text-center text-2xl font-medium2">
+                {lnurlSuccessAction.message}
+              </Text>
+            }
+            {lnurlSuccessAction.tag == "url" &&
```

Comment:
is this correct? you have a same inner check too

Reply:
The `tag` defines which type the successAction has. If the tag is "url", we use the `url` field from the SuccessAction. :thinking: